Resources/Prototypes/Entities/Structures/Machines/vending_machines.yml

```diff
@@ -547,8 +547,8 @@
 - type: entity
   parent: VendingMachine
   id: VendingMachineSmartFridge
-  name: SmartFridge
-  description: A refrigerated storage unit for storing medicine and chemicals.
+  name: Smart Fridge
+  description: A refrigerated storage unit for keeping items cold and fresh.
```

Comment:
why change the description?

Reply:
The smart fridge doesn't just hold medicine or chemicals. It's also used in the kitchen and in a few other places.

Reply:
true